src/themes/default/elements/icon.overrides

```diff
@@ -684,6 +698,23 @@ i.icon.fort.awesome:before { content: "\f286"; }
 i.icon.product.hunt:before { content: "\f288"; }
 i.icon.mixcloud:before { content: "\f289"; }
 i.icon.scribd:before { content: "\f28a"; }
+i.icon.gitlab:before { content: "\f296"; }
+i.icon.wpbeginner:before { content: "\f297"; }
+i.icon.wpforms:before { content: "\f298"; }
+i.icon.envira.gallery:before { content: "\f299"; }
+i.icon.glide:before { content: "\f2a5"; }
+i.icon.glide.g:before { content: "\f2a6"; }
+i.icon.viadeo:before { content: "\f2a9"; }
+i.icon.viadeo.square:before { content: "\f2aa"; }
+i.icon.snapchat:before { content: "\f2ab"; }
+i.icon.snapchat.ghost:before { content: "\f2ac"; }
+i.icon.snapchat.square:before { content: "\f2ad"; }
+i.icon.pied.piper.hat:before { content: "\f2ae"; }
```

Comment:
New icon: `pied piper hat`, the new brand logo for the compression solution. The `alternative pied piper` icon **is officialy deprecated**, as noted in their [website](http://www.piedpiper.com/), *"The old Pied Piper “flute player” logo was referencing the fairy-tale character and in no way was meant to suggest a sexual act involving either the mouth or ear of the figure depicted."*